TS7016:  Could not find a declaration file for module 'single-spa-vue'

I am getting the following error in a Vue + Vite project:

>  error TS7016: Could not find a declaration file for module 'single-spa-vue'. 'C:/Users/xxx/src/vue-mife01/node_modules/single-spa-vue/dist/esm/single-spa-vue.js' implicitly has an 'any' type.
>
> There are types at 'C:/Users/xxx/src/vue-mife01/node_modules/single-spa-vue/types/single-spa-vue.d.ts', but this result could not be resolved when respecting package.json "exports". The 'single-spa-vue' library may need to update its package.json or typings.

This error prevents building the application.

I found a workaround:

1. Create a new folder in the root of the project.  Name it something like `typings`, or `custom-typings`.
2. Create a sub-folder named `single-spa-vue`.
3. Copy into this sub-folder the file `node_modules/single-spa-vue/types/single-spa-vue.d.ts` and rename it to `index.d.ts`.
4. Edit the project's `tsconfig.json` file to include the `paths` property under `compilerOptions` to include this custom typings folder.  Something like this:

```json
{
  "compilerOptions": {
    "paths": {
      "*": [ "./custom-typings/*" ]
    },
    ...
  }
}
```

Unrelated question:  Is this project abandoned?  No updates for 2 years.

--- a/package.json
+++ b/package.json
@@ -6,6 +6,7 @@
   "module": "dist/esm/single-spa-vue.js",
   "exports": {
     ".": {
+      "types": "./types/single-spa-vue.d.ts",
       "import": "./dist/esm/single-spa-vue.js",
       "require": "./dist/umd/single-spa-vue.js"
     },

